velox/exec/GroupingSet.cpp

```diff
@@ -158,11 +130,10 @@ void GroupingSet::addInput(const RowVectorPtr& input, bool mayPushdown) {
     addInput(input, mayPushdown);
     return;
   }
-  numAdded_ += lookup_->rows.size();
```

Comment:
Just curious : I can understand why numAdded_ could be removed in the globalAggregation code-path. But I'm not sure what this line was meant to signify. Please can you explain.

Reply:
numAdded_ was used only in global aggregation code path. It is not needed and wasn't used in group-by code paths.